### PR TITLE
Stochasticity: harmonise variable names, clarify definitions and fix typos

### PR DESCRIPTION
**Pull request by pwinskill**
A few small changes to the stochasticity practical - nothing functional, all clarifications, renaming, comments and typos

### CODE REVIEW

File: help/stochasticity-2.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-## Example 2: Stochastic growth and death model
+## Example 2: Stochastic birth and death model
 
-This model includes births and deaths at per-capita rates $r$ and $s$ per week, respectively. The ODE for $N$ is made up of the difference between the mean growth rate and the mean death rate:
+This model includes births and deaths at per-capita rates $r$ and $mu$ per week, respectively. The ODE for $N$ is made up of the difference between the mean birth rate and the mean death rate:
 
-$$ \frac{dN}{dt}=rN-sN $$
+$$ \frac{dN}{dt}=rN-muN $$
```
Comment (lwhittles) [inline help/stochasticity-2.md:5]: Does this render correctly? I reckon you need a backslash
```suggestion
$$ \frac{dN}{dt}=rN-\mu N $$
```

File: help/stochasticity-2.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-The difference $(r-s)$ is the net growth rate and the solution is either exponential growth or decline i.e.,
+The difference $(r-mu)$ is the net birth rate and the solution is either exponential growth or decline i.e.,
```
Comment (lwhittles) [inline help/stochasticity-2.md:7]:
```suggestion
The difference $(r-\mu)$ is the net birth rate and the solution is either exponential growth or decline i.e.,
```

File: help/stochasticity-2.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 
-$$ N(t) =N(0) \exp((r-s)t) $$
+$$ N(t) =N(0) \exp((r-mu)t) $$
```
Comment (lwhittles) [inline help/stochasticity-2.md:9]:
```suggestion
$$ N(t) =N(0) \exp((r-\mu)t) $$
```

File: help/stochasticity-2.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 
-If births exceed deaths ($r > s$) the population grows. If deaths exceed births ($s > r$) the population shrinks.
+If births exceed deaths ($r > mu$) the population grows. If deaths exceed births ($mu > r$) the population shrinks.
```
Comment (lwhittles) [inline help/stochasticity-2.md:11]:
```suggestion
If births exceed deaths ($r > \mu$) the population grows. If deaths exceed births ($mu > r$) the population shrinks.
```

File: help/stochasticity-2.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
 
 In the stochastic simulation, we have to be more careful as we are dealing with individuals. Each individual has a probability of reproducing and of dying in some short interval, $dt$. However, it can’t die and then give birth! These two dependent events that can happen are known as competing hazards. To calculate the number of births and deaths in a short time, we first calculate the number of births or deaths:
 
-$$ \mbox{Births or deaths in }dt, n \sim \mathrm{Binomial}(N,(r + s)dt) $$
+$$ \mbox{Births or deaths in }dt, n \sim \mathrm{Binomial}(N,(r + mu)dt) $$
```
Comment (lwhittles) [inline help/stochasticity-2.md:15]:
```suggestion
$$ \mbox{Births or deaths in }dt, n \sim \mathrm{Binomial}(N,(r + \mu)dt) $$
```

File: help/stochasticity-2.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
 
 We then decide which were births and which deaths:
 
-$$ \mbox{Deaths in }dt \sim \mathrm{Binomial}(s / (r + s),n) $$
+$$ \mbox{Deaths in }dt \sim \mathrm{Binomial}(mu / (r + mu),n) $$
```
Comment (lwhittles) [inline help/stochasticity-2.md:19]:
```suggestion
$$ \mbox{Deaths in }dt \sim \mathrm{Binomial}(mu / (r + \mu),n) $$
```

**Review by lwhittles — CHANGES_REQUESTED** [review]
Might be mistaken but my mental latex compiler reckons you want a couple of backslashes

**Comment by pwinskill** [comment]
Good spot. Thanks @lwhittles

**Pull request merged**